test/clj/rems/test_layers.clj

```diff
@@ -26,11 +26,6 @@
            (str/starts-with? s "rems.service.")
            (str/starts-with? s "rems.api"))))
 
-(defn rename-ns [s]
-  ;; TODO implement these namespace renames
-  (-> s
-      (str/replace "rems.db.test-data" "rems.service.test-data")))
-
```

Comment:
We could still keep the function for future renames as we progress with the architecture tool. Although, we can of course bring it back from git history.

Reply:
let's keep the function for now, it's easier to modify when present in the code (easy to skip by adding reader tags).